### ISSUE TITLE
[BUG]: Translate from another locale Link

### ISSUE THREAD

**Issue by MarcosVVMK** [Fekide/strapi-plugin-translate#349]
**Describe the bug**

I found a plugin bug that occurs in strapi versions above 4.11.7, in the part where the "Translate from another locale" link of the collection type should appear, it only appears after the screen is reloaded.

**To Reproduce**

1. Install the strapi translate plugin in a strapi project with a version above v4.11.7
2. Add at least 1 more locale
3. Access a post of any collection type
4. Click on the locales dropdown and select another language for translation
5. The Translate from another locale button does not appear unless the page is reloaded

**Expected behavior**
The button with the text "Translate from another locale" should appear without having to reload the page

**Screenshots**

https://imgur.com/a/54WBVYo

**System (please complete the following information):**

- OS: Windows (WSL2: Ubuntu 22.04)
- Browser Google Chrome
- Versions: 
  - strapi: v4.14.4
  - strapi-plugin-translate: v1.2.2
  - provider: Customprovider file

**Plugin config**
Please add your plugin config here, e.g.

```js
 translate: {
    enabled: true,
    config: {
      provider: 'chatgpt',
      providerOptions: {
      },
      translatedFieldTypes: [
        'string',
        { type: 'text', format: 'plain' },
        { type: 'richtext', format: 'markdown' },
        'component',
        'dynamiczone',
      ],
      // If relations should be translated (default true)
      translateRelations: false,
    },
  },
```

**Comment by sargreal** [comment]
Hi @MarcosVVMK and thanks for opening this issue.

I have tested a bit and found that "Fill in from another locale link" does also not work in the i18n Plugin where much of this plugin's code is based on. I have created an upstream issue which addresses the root cause: https://github.com/strapi/strapi/issues/18601

**Comment by sargreal** [comment]
This issue has most likely been solved in Strapi v4.19.1, at least I don't see the issue anymore
